src/test/java/org/apache/sysds/test/TestUtils.java

```diff
@@ -1802,6 +1802,16 @@ public static MatrixBlock generateTestMatrixBlock(int rows, int cols, double min
 		return MatrixBlock.randOperations(rows, cols, sparsity, min, max, "Uniform", seed);
 	}
 
+	public static MatrixBlock generateTestMatrixBlockSym(int rows, int cols, double min, double max, double sparsity, long seed){
```

Comment:
add java doc comment here, to make it easy to find this method,(add keywords or something) we have a bunch of tests using symmetric matrices and all implement their own, and it would be nice if everyone use the same generator (this could be it!).